include/eve/module/core/regular/impl/simd/arm/sve/any.hpp

```diff
@@ -0,0 +1,31 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/has_abi.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
+{
+template<scalar_value T, typename N, relative_conditional_expr C>
+EVE_FORCEINLINE bool
+any_(EVE_SUPPORTS(sve_), C const& cond, logical<wide<T,N>> const& v) noexcept
+requires sve_abi<abi_t<T, N>>
+{
+  if constexpr( C::is_complete && !C::is_inverted ) return false;
```

Comment:
Test for ignore none here and remove the other overload:

https://github.com/jfalcou/eve/blob/8540c2bee2ec485f3b09fe9df83ac794417c5319/include/eve/module/core/regular/impl/any.hpp#L61